addons/website/views/snippets/s_timeline.xml

```diff
@@ -2,51 +2,58 @@
 <odoo>
 
 <template name="Timeline" id="s_timeline">
-    <section class="s_timeline pt24 pb48">
-        <div class="container s_timeline_line">
-            <div class="s_timeline_row d-block d-md-flex flex-row" data-name="Row">
-                <div class="s_timeline_date"><b>2019</b></div>
-                <div class="s_timeline_content d-flex">
-                    <div class="s_timeline_card s_card card bg-white w-100" data-name="Card" data-snippet="s_card">
-                        <div class="card-body">
-                            <h5 class="card-title">Your title</h5>
-                            <p class="card-text">A timeline is a graphical representation on which important events are marked.</p>
-                        </div>
-                    </div>
-                    <i class="fa fa-1x fa-child bg-o-color-2 rounded-circle s_timeline_icon"/>
+    <section class="s_timeline pt48 pb48" data-vcss="001">
+        <div class="o_container_small">
+            <div class="row">
+                <div class="col-12 pb24 text-center" data-name="Heading">
+                    <h2 class="h3-fs">Latest news</h2>
+                    <p class="lead">Highlight your history, showcase growth and key milestones.</p>
                 </div>
-                <div class="s_timeline_content"/>
             </div>
-            <div class="s_timeline_row d-block d-md-flex flex-row" data-name="Row">
-                <div class="s_timeline_date"><b>2018</b></div>
-                <div class="s_timeline_content d-flex">
-                    <div class="s_timeline_card s_card card bg-white w-100" data-name="Card" data-snippet="s_card">
-                        <div class="card-body">
-                            <p class="card-text">You can edit, duplicate...</p>
+            <div class="position-relative">
+                <div class="s_timeline_line position-absolute top-0 bottom-0 d-block border-start"/>
+                <div class="s_timeline_row position-relative d-flex gap-5 align-items-start pb-4" data-name="Year">
```

Comment:
Nitpick: if you manage to find another place for the `s_timeline_line` element, then maybe set a `pt-4` class instead of `pb-4` (or maybe `pt-2` and `pb-2`) so there is a bit of padding before the dot (so the overlay looks better) :slightly_smiling_face: 
![image](https://github.com/user-attachments/assets/260084a3-1cfa-4d70-84be-397157ec4267)

Of course, do it only if you see that the line doesn't go above the dot, like here :arrow_down:  :wink: 
![image](https://github.com/user-attachments/assets/7be31e26-e547-42b4-b660-1c11b8a398bd)